app/scenes/Document/Document.js

```diff
@@ -128,6 +129,11 @@ class DocumentScene extends Component {
     }
   };
 
+  loadEditor = async () => {
+    const EditorImport = await import('components/Editor');
```

Comment:
I actually liked `react-loadable` but was easier as we want to share the loading state with document loading indicator